website/components/remote-plugin-docs/utils/validate-plugin-docs-files.js

```diff
@@ -13,15 +13,22 @@ const COMPONENT_TYPES = [
 // with at least one .mdx file within it.
 function validatePluginDocsFiles(filePaths) {
   function isValidPath(filePath) {
+    // Allow the docs root folder
     const isDocsRoot = filePath === 'docs/'
+    // Allow component folders
     const isComponentRoot = COMPONENT_TYPES.reduce((acc, type) => {
       return acc || filePath === `docs/${type}/`
     }, false)
+    // Allow .mdx files in component folders
     const isComponentMdx = COMPONENT_TYPES.reduce((acc, type) => {
       const mdxPathRegex = new RegExp(`^docs/${type}/(.*).mdx$`)
       return acc || mdxPathRegex.test(filePath)
     }, false)
-    const isValidPath = isDocsRoot || isComponentRoot || isComponentMdx
+    // Allow docs/README.md files
```

Comment:
A `README.md` file is now allowed in plugin `docs` folders, rather than causing our build process to throw a validation error. Note that this file is _not_ included in our build - we still only include `.mdx` files.